setup/create_db.pl

```diff
@@ -21,6 +21,9 @@
 use strict;
 use DBI();
 use Getopt::Long;
+use Crypt::Eksblowfish::Bcrypt qw(bcrypt);
+use Crypt::OpenSSL::Random;
+use MIME::Base64 ();
```

Comment:
Speaking of debian this Eksblowfish and Openssl-Libraries for perl can be installed via packet manager. I didn't find a proper way with standard-functions (I can only copy&paste-perl) and you shouldn't write your own crypto. Like this they are working and generating hashes that our vexim-scripts can handle.